LICENSE

```diff
@@ -22,3 +26,29 @@ IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
 CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
 TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+"""
+
+This license applies to tools/update-authors.sh which is based on
+update-authors.sh from https://github.com/nodejs/node:
+
+"""
+Copyright Node.js contributors. All rights reserved.
```

Comment:
@hacksparrow The script is covered by this license. I did the simple thing and just included it. Maybe we can do something more condensed to cover any files that originate externally and have a line for <file> <copyright> <license type> as a more lightweight summary though.